tests/test_package/test_convert.py

```diff
@@ -0,0 +1,16 @@
+from tidy3d.web.cli import tidy3d_cli
+from tidy3d.web.cli.converter import converter_arg
+from pathlib import Path
+
+
+def test_tidy3d_converter():
+    lsf_path = str(Path.cwd()) + "/tests/data/example.lsf"
+    converter_arg(lsf_path, "lsf_converter_test.py")
+    file_path = Path("lsf_converter_test.py")
```

Comment:
We've been using temporary files for things like this in tests, which helps the tests be faster and avoids unnecessary file creation bugs. There is a [tmp_path](https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html) tool in pytest for this, see example [here](https://github.com/flexcompute/tidy3d/blob/develop/tests/test_components/test_IO.py#L65-L69)

When you pass `tmp_path` as an argument, you can basically write to it and it will be handled virtually. Let's do something similar here for consistency with the rest of the tests.